### DISCUSSION TITLE
Compilação

### DISCUSSION THREAD

**Discussion by Mr-Noob117** [aircrack-ng/aircrack-ng#2581] [General]
Eu tenho uma placa de rede TL-WN722N v2/v3  Driver 8188eu. Estou usando o kernel 6.5.0-kali1-amd64. Até qual versão tem suporte o aircrack-ng??? eu tento baixar um headers e recebo essa mensagem:

E: Impossível encontrar o pacote linux-headers-$6.5.0-kali1-amd64
E: Couldn't find any package by glob 'linux-headers-$6.5.0-kali1-amd64'
E: Não foi possível encontrar o pacote através da expressão regular linux-headers-$`uname -r`

o script que eu usava era esse:

apt update
apt install bc
apt-get install build-essential
apt-get install libelf-dev
apt-get install linux-headers-$`uname -r`
apt install dkms

REINICIE O LINUX

rmmod r8188eu.ko
git clone https://github.com/aircrack-ng/rtl8188eus
cd rtl8188eus
sudo -i
echo "blacklist r8188eu" > "/etc/modprobe.d/realtek.conf"
echo 'blacklist r8188eu'|sudo tee -a '/etc/modprobe.d/realtek.conf'
exit
make
make install

mas sem instalar os headers parece que nao vai ser possivel!!!!

Alguém pode me ajudar???

**Comment by RaspBoss** [comment]
mesmo problema com a 8812au, vou fazer um downgrade na versão e tentar a instalação novamente.
